admin/ai/openai.go

```diff
@@ -34,7 +34,7 @@ func (c *openAI) Complete(ctx context.Context, msgs []*adminv1.CompletionMessage
 	}
 
 	res, err := c.client.CreateChatCompletion(ctx, openai.ChatCompletionRequest{
-		Model:       openai.GPT3Dot5Turbo,
+		Model:       openai.GPT4o,
```

Comment:
When using GPT4o in chats, I have noticed that its unavailable many times. I wonder if we should have a list of models for fallback ? i.e. If gpt4o isn't available fallback to 3.5 turbo ?
@begelundmuller : Any thoughts ?